packages/jsapi-utils/src/MessageUtils.ts

```diff
@@ -10,17 +10,69 @@ export const LOGIN_OPTIONS_REQUEST =
 export const SESSION_DETAILS_REQUEST =
   'io.deephaven.message.SessionDetails.request';
 
+/**
+ * Use a BroadcastChannel for sending messages between tabs, such as when the user logs out.
+ * Something that isn't clear from the docs - for inter-browser communication, you must use `postMessage` and `onmessage`.
+ * `BroadcastChannel` is an `EventTarget`, and does support `addEventListener`/`dispatchEvent` etc... but that's only within the same instance of that object!
+ * https://developer.mozilla.org/en-US/docs/Web/API/Broadcast_Channel_API
+ */
```

Comment:
Yea I like that for themes/user preferences. It might be able to work for this as well, I was just porting over the existing functionality we've got from Enterprise.